cpp/oneapi/dal/algo/linear_regression/common.cpp

```diff
@@ -42,6 +42,8 @@ class descriptor_impl : public base {
     explicit descriptor_impl() = default;
 
     bool compute_intercept = true;
+    double alpha = 0;
```

Comment:
probably, its better to order it in all files: common.cpp and common.hpp in the same way(and get/set methods)